packages/models-library/src/models_library/api_schemas_webserver/storage.py

```diff
@@ -41,10 +38,4 @@ class BatchDeletePathsBodyParams(InputSchema):
 
 
 class DataExportPost(InputSchema):
-    paths: list[StorageFileID]
-
-    def to_rpc_schema(self, location_id: LocationID) -> DataExportTaskStartInput:
-        return DataExportTaskStartInput(
-            file_and_folder_ids=self.paths,
-            location_id=location_id,
-        )
+    paths: list[str]
```

Comment:
Please use Path instead of str (similar to delete above, or path size computation)